---
title: "Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex"
date: 2014-08-01
publishDate: 2023-04-26T19:37:47.351795Z
authors: ["Alex A Pollen", "Tomasz J Nowakowski", "Joe Shuga", "Xiaohui Wang", "Anne A Leyrat", "Jan H Lui", "Nianzhen Li", "Lukasz Szpankowski", "Brian Fowler", "Peilin Chen", "Naveen Ramalingam", "Gang Sun", "Myo Thu", "Michael Norris", "Ronald Lebofsky", "Dominique Toppani", "2nd, Darnell W Kemp", "Michael Wong", "Barry Clerkson", "Brittnee N Jones", "Shiquan Wu", "Lawrence Knutsson", "Beatriz Alvarado", "Jing Wang", "Lesley S Weaver", "Andrew P May", "Robert C Jones", "Marc A Unger", "Arnold R Kriegstein", "Jay A A West"]
publication_types: ["2"]
abstract: "Large-scale surveys of single-cell gene expression have the potential to reveal rare cell populations and lineage relationships but require efficient methods for cell capture and mRNA sequencing. Although cellular barcoding strategies allow parallel sequencing of single cells at ultra-low depths, the limitations of shallow sequencing have not been investigated directly. By capturing 301 single cells from 11 populations using microfluidics and analyzing single-cell transcriptomes across downsampled sequencing depths, we demonstrate that shallow single-cell mRNA sequencing (~50,000 reads per cell) is sufficient for unbiased cell-type classification and biomarker identification. In the developing cortex, we identify diverse cell types, including multiple progenitor and neuronal subtypes, and we identify EGR1 and FOS as previously unreported candidate targets of Notch signaling in human but not mouse radial glia. Our strategy establishes an efficient method for unbiased analysis and comparison of cell populations from heterogeneous tissue by microfluidic single-cell capture and low-coverage sequencing of many cells."
featured: false
publication: "*Nat Biotechnol*"
---

